src/wayland/shell/wlr_layer/mod.rs

```diff
@@ -246,6 +246,9 @@ pub trait WlrLayerShellHandler {
 
     /// A surface has acknowledged a configure serial.
     fn ack_configure(&mut self, surface: wl_surface::WlSurface, configure: LayerSurfaceConfigure) {}
+
+    /// A layer surface was destroyed.
+    fn destroyed(&mut self) {}
```

Comment:
Might be useful to know what surface was destroyed.

Reply:
I was actually going to do that but then noticed it isn't passed to the destroyed handler, but looks like it's part of the surface data. Should be easy enough.